conn.go

```diff
@@ -25,11 +25,16 @@ type ConnectionState struct {
 }
 
 type Conn struct {
-	conn       net.Conn
-	text       *textproto.Conn
-	server     *Server
-	helo       string
-	nbrErrors  int
+	conn   net.Conn
+	text   *textproto.Conn
+	server *Server
+	helo   string
+
+	// Number of errors witnessed on this connection
+	errCount int
+	// Number of errors we'll tolerate per connection before closing. Defaults to 3.
+	errThreshold int
```

Comment:
This can be a global constant

Reply:
For sure. My thinking was that it could be configurable if that is wanted in the future, but I can move it out into a global for now.